File: src/net/pryoscode/CustomDiscordRPC/menu/items/Exit.java
package net.pryoscode.CustomDiscordRPC.menu.items;

import net.pryoscode.CustomDiscordRPC.Main;
import java.awt.*;
import java.awt.event.ActionEvent;
import java.awt.event.ActionListener;

public class Exit extends MenuItem {

    public Exit() {
        setLabel("Exit");
        addActionListener(new ActionListener() {
            @Override
            public void actionPerformed(ActionEvent e) {
                Main.trayUtils.removeTrayIcon();
                System.exit(0);
            }
        });
    }

}

File: src/main/kotlin/net/pryoscode/customdiscordrpc/menu/items/Exit.kt
package net.pryoscode.customdiscordrpc

import java.awt.MenuItem
import java.awt.event.ActionListener
import java.awt.event.ActionEvent

class Exit : MenuItem, ActionListener {

    constructor() {
        setLabel("Exit")
        addActionListener(this)
    }

    override fun actionPerformed(event: ActionEvent) {
        System.exit(0)
    }

}